tests/config.spec.ts

```diff
@@ -0,0 +1,26 @@
+import * as m from 'mochainon';
+import * as path from 'path';
+
+import config from '../lib/config';
+
+// tslint:disable no-unused-expression
```

Comment:
I tried to keep things the same as much as possible to validate my changes, but I personally like explicit calls more too